### ISSUE TITLE
Oh boy to choose video-transcode or other-transcode - doing x264 to HEVC 10 bit - 1660 ti nvenc maybe

### ISSUE THREAD

**Issue by mshiels** [donmelton/other_video_transcoding#131]
So the question is which to use for best quality, my understanding is Handbrake x265 is somehow modified/not as good as ffpmeg x265 if that right?    Just wondering if without digesting all the code it's easy to say one is better than the other at doing a 8bit x264 or remux to x265 10 bit encode.   

Ideally trying to use nvenc but I'm running 2012 R2 server which seems to have no good drives anymore.  So about to rebuild my boot drive for 2019 server maybe.  Otherwise the nvenc just barfs on driver version problems.

Going to be taking my massive Plex library and encoding it to save some space but also ideally set it up for slightly better quality in some cases doing some 4K to 1080p encodes where the x265 10bit will help.

**Comment by donmelton** [comment]
@mshiels Thank you for using my tools.

I assume you're asking which tool you should use to transcode input in 8-bit H.264 format to output in 10-bit HEVC format? Your terminology is a bit confusing because `x264` and `x265` are the names of software-based encoders, not formats themselves.

This simple answer is that you should use `other-transcode` to do this. But not because `ffmpeg` is somehow better than `HandBrakeCLI`. Rather, it's because my older `transcode-video` tool is no longer in active development and because my newer `other-transcode` tool is specifically designed to work well with Nvidia encoders.

Does that answer your question?

**Comment by mshiels** [comment]
Yup, cool, and thanks!!  But sorry to say I am more adjusting my tools based on what I can find.

 

I have a similar engine to your encoder in C# for .Net – but it’s kinda on steroids

*	It runs as a post processor to plex recordings and web downloads etc. 
*	It demuxes any built in subtitles in the video stream using CCExtractor
*	It merges in any subtitles it finds in the original directory
*	OCRs any PGS/VobSub to SRT using SUbtitleEdit
*	Downloads using subliminal any possible Subtitles
*	Copies all audio, and makes a AC3 track since my stupid old tv can’t even do eac3 
*	And yes I ‘read’ a lot of my tv – late at night, so don’t want to bother the wife. 
*	And a lot of foreign stuff too.  
*	AND Now maybe getting adjusted to do some X265/nvenc encoding, seems storage space is ‘never enough’. 
*
